grafeas-v1/snippets/snippet_metadata_grafeas.v1.json

```diff
@@ -1,7 +1,7 @@
 {
   "client_library": {
     "name": "grafeas-v1",
-    "version": "0.9.0",
+    "version": "0.10.0",
```

Comment:
why is this change needed ?

Reply:
Not sure how this change became part of the PR. Removed this.